openmdao/devtools/partition_tree_n2.py

```diff
@@ -36,17 +44,32 @@ def _tree_dict(ss, component_execution_orders, component_execution_index):
         children = [_tree_dict(s, component_execution_orders, component_execution_index) for s in ss.subsystems()]
 
         if isinstance(ss, Component):
-            for vname, meta in ss.params.items():
-                dtype=type(meta['val']).__name__
-                children.append(OrderedDict([('name', vname), ('type', 'param'), ('dtype', dtype)]))
-
-            for vname, meta in ss.unknowns.items():
-                dtype=type(meta['val']).__name__
-                implicit = False
-                if meta.get('state'):
-                    implicit = True
-                children.append(OrderedDict([('name', vname), ('type', 'unknown'), ('implicit', implicit), ('dtype', dtype)]))
-
+            if ss.is_active():
+                my_chlist = []
+                for vname, meta in ss.params.items():
+                    dtype=type(meta['val']).__name__
+                    my_chlist.append(OrderedDict([('name', vname), ('type', 'param'), ('dtype', dtype)]))
```

Comment:
yes otherwise json order is random and this makes it nice to diff and see that everything is exactly the same on successive runs